From 3b7a5b7dbe2a3c2d03390a1ed3348ea6616538e0 Mon Sep 17 00:00:00 2001
From: "A. Maitland Bottoms" <bottoms@debian.org>
Date: Fri, 26 Oct 2018 21:13:38 -0400
Subject: [PATCH 2/6] improve UHD_IMAGES_DIR logic

A CMake option is either On or Off.
UHD_IMAGES_DIR gets a sane default if not specified on the
CMake command line.
---
 host/CMakeLists.txt | 3 ---
 1 file changed, 3 deletions(-)

diff --git a/host/CMakeLists.txt b/host/CMakeLists.txt
index 281c55c5d..b82306f7a 100644
--- a/host/CMakeLists.txt
+++ b/host/CMakeLists.txt
@@ -166,10 +166,7 @@ IF(NOT DEFINED UHD_IMAGES_DIR)
         SET(UHD_IMAGES_DIR ${using_images_dir})
     ENDIF(DEFINED FPGA_IMAGES_DIR)
 ENDIF(NOT DEFINED UHD_IMAGES_DIR)
-
-OPTION(UHD_IMAGES_DIR "Path to installed UHD image binaries.")
 MESSAGE( STATUS "Using UHD Images Directory: ${UHD_IMAGES_DIR}" )
-ADD_DEFINITIONS(-DUHD_IMAGES_DIR=${UHD_IMAGES_DIR})
 
 IF(DEFINED UHD_IMAGES_DIR_WINREG_KEY)
     ADD_DEFINITIONS(-DUHD_IMAGES_DIR_WINREG_KEY=${UHD_IMAGES_DIR_WINREG_KEY})
-- 
2.11.0

